docs/architecture.md

```diff
@@ -18,3 +18,94 @@
   subscribe to the Redux store without React, but this would be more complicated.
 
 ## Back-end architecture
+
+![Diagram](images/architecture-back-end.drawio.svg)
+
+### Dataset files
+
+All persistent data is stored on the back-end server as JSON files, in the following folder structure
+as seen from the SERVER_DATA_ROOT location:
+
+```
+├── datasets
+│   ├── some-dataset
+│   │   ├── locations.json (array of lng-lat coordinates for each initiative)
+│   │   ├── searchable.json (array of the property values and searchable strings for each initiative)
+│   │   ├── initiatives
+│   │   |   ├── 0.json (full info of first initiative in the above aggregate JSONs)
+│   │   |   ├── 1.json
+│   │   |   ├── ...
+│   ├── other-dataset
+│   │   ├── ...
+│   ├── ...
+```
+
+Additionally, for each map variant, there's a `config.json`, which contains config for displaying the map in the UI, including the vocabs (translations of data IDs), default sidebar panel, and popup appearance.
```

Comment:
I thought `config` would also code UI related things like the old `config.ts`, and would need to be checked into the source code, rather than generated by a script like the actual data. But I think it's maybe fine to have it per-dataset rather than per-map. You're right though, these docs imply configs aren't associated with a specific dataset. Maybe they don't need to be if we want to share datasets in the future but I think it over-complicates things for now, so it's better to have one config per dataset and stick with linking them to a datasetID

Reply:
I've updated these docs to reflect my above comment, which isn't very clear. And I've also updated the folder structure in PR #24 to match this

Reply:
I think the datasets (and configs) could be version controlled, and in fact I have been doing that - I just checked the whole datastore into git. Would we want the config to be version controlled separately? Not sure, I think it might not need to be though.